test/e2e/e2e_test.go

```diff
@@ -16,77 +16,77 @@ limitations under the License.
 
 package e2e
```

Comment:
Can we just completely remove the `test/e2e` directory here?
I've got a PR in-flight for new e2es: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/212

Reply:
SGTM!